internal/transport/transport.go

```diff
@@ -566,6 +567,8 @@ type ConnectOptions struct {
 	ChannelzParentID int64
 	// MaxHeaderListSize sets the max (uncompressed) size of header list that is prepared to be received.
 	MaxHeaderListSize *uint32
+	// MaxFrameSize sets the max size of frame size
```

Comment:
`// MaxFrameSize sets the max size of an HTTP/2 frame.`

Reply:
done